components/layout/app-bar.jsx

```diff
@@ -1,8 +1,8 @@
-import React, { useContext } from 'react'
+import React from 'react'
 import { AppBar } from '@oacore/design'
 import { classNames } from '@oacore/design/lib/utils'
 
-import LayoutContext from './context'
+import useSidebar from './use-sidebar'
```

Comment:
Would `hooks.js` satisfy you? The interface will remain similar, however, the consistency would not be exactly the same. I did not want to do it in that way because it required a rule disabling what I did not want to.

I would prefer simplicity over consistency. Also, I am afraid this component will change one more time. I am not very satisfied with it.

Reply:
Hooks.js is fine too. Out of curiosity what rule you would need to disable? :) 

Thanks

Reply:
`import/prefer-default-export`